bin/components/page-component/template.pug

```diff
@@ -109,7 +120,6 @@ html
     script(src=relative.root + 'assets/scripts/h3.js' charset='UTF-8')
     script(src=relative.root + 'assets/scripts/iframe.js' charset='UTF-8')
     script(src=relative.root + 'assets/scripts/fullscreen.js' charset='UTF-8')
-    script(src=relative.root + 'assets/scripts/git.js' charset='UTF-8')
```

Comment:
Branches moet dynamisch opgebouwd worden op basis van branches.js, zie ook regel  109 (script(src=relative.basePath + 'branches.js' charset='UTF-8'))

Dit kan niet weg. Het menu wordt dynamisch opgebouwd op basis van branches.json in de root van het project. Daar staat altijd de meest actuele verzameling van branches. Omdat het branches menu dynamisch wordt opgebouwd vanuit dit bestand is het branches menu hetzelfde over alle features.

Reply:
Ik heb git.js teruggezet en aangepast.